snapshot/Program.cs

```diff
@@ -0,0 +1,133 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace Dapr.Tests.Snapshot
+{
+    using Dapr.Actors;
+    using Dapr.Actors.Client;
+    using Dapr.Client;
+    using Dapr.Tests.HashTagApp.Actors;
+    using Microsoft.AspNetCore.Hosting;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.Extensions.Hosting;
+    using Microsoft.Extensions.Logging;
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Threading.Tasks;
+
+    public class Program
+    {
+        // This uses the names of shapes for a generic theme
+        static internal string[] HashTags = new string[]
```

Comment:
this is also in https://github.com/dapr/test-infra/blob/master/feed-generator/Program.cs
can be moved to a common constants or file can be source included.

Reply:
Yes I agree, but it's extra work to get the images building if there's a common folder outside.  I'd like to do that later.

Reply:
All the common files can be moved to a common .dll or can be source included from a common folder

Reply:
Opened https://github.com/dapr/test-infra/issues/47